app/ide-desktop/lib/dashboard/src/utilities/event.ts

```diff
@@ -50,3 +50,13 @@ export function isTextInputEvent(event: KeyboardEvent | React.KeyboardEvent) {
   // Allow `alt` key to be pressed in case it is being used to enter special characters.
   return !event.ctrlKey && !event.shiftKey && !event.metaKey && isTextInputKey(event)
 }
+
+/** Whether the element accepts keyboard events. */
```

Comment:
I think here we should write smth like `Whether the element accepts user inputs`, cause many elements accepts keyboard events, for example - select, button, a, many more :)